### DISCUSSION TITLE
HTML formatting

### DISCUSSION THREAD

**Discussion by eseiver** [jdepoix/youtube-transcript-api#191] [Q&A]
Hello, I was wondering if it's possible to preserve HTML formatting from manual captions when downloading them. The subtitles I download have meaningful `<i>` and `<b>` labeling for some words, but they're not appearing in the downloaded captions. Is there a setting I can toggle?
Thanks!!

**Comment by eseiver** [comment]
Addressed in #192.

**Comment by jdepoix** [comment]
This has been implemented by @eseiver and released in [v0.6.0](https://github.com/jdepoix/youtube-transcript-api/releases/tag/v0.6.0). Thank you for your contribution @eseiver ! 😊 🙏
